src/core/Akka.Tests/Actor/FunctionRefSpec.cs

```diff
@@ -0,0 +1,253 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="FunctionRefSpec.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2018 Lightbend Inc. <http://www.lightbend.com>
+//     Copyright (C) 2018 Akka.NET project <https://github.com/akkadotnet/akka.net>
```

Comment:
needs the year range for Akka.NET too.

Reply:
@marcpiechura going to be submitting an update to all of the headers soon